### ISSUE TITLE
Accessibility based cache deletion fails on HyperOS 2.0

### ISSUE THREAD

**Issue by gianpiero29** [d4rken-org/sdmaid-se#1538]
App cleaner does not work on my Redmi Note 13 since I upgraded to HyperOs 2.0

**Comment by d4rken** [comment]
Please provide a debug log

**Comment by d4rken** [comment]
If you just tried to attach a file via email, this does unfortunately not work.

**Comment by d4rken** [comment]
Closing this in favor of #1521

**Comment by d4rken** [comment]
Re-opening this because the other issue's ROM is still MIUIv14, not HyperOS. Sorry.
